packages/viewer/src/components/viewer-spinner/viewer-spinner.css

```diff
@@ -0,0 +1,94 @@
+
+
+.spinner {
+  position: relative;
+  display: flex;
+  height: 16px;
+  width: 16px;
+}
+
+.spinner div {
+  position: absolute;
+  display: block;
+  animation: spinner 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
+  border-radius: 50%;
+  border-style: solid;
+  box-sizing: border-box;
+  border-color: var(--neutral-800) transparent transparent transparent;
+  border-width: var(--border-width, 1px);
+
+  margin: 0;
+}
+
+.spinner div:nth-child(1) {
+  animation-delay: -0.45s;
+}
+
+.spinner div:nth-child(2) {
+  animation-delay: -0.3s;
+}
+
+.spinner div:nth-child(3) {
+  animation-delay: -0.15s;
+}
+
+/* xs */
+.spinner.xs {
+  height: 16px;
+  width: 16px;
+}
+
+.spinner.xs div {
+  border-width: var(--border-width, 1px);
+  height: 16px;
+  width: 16px;
+  margin: 0;
+}
+
+/* sm */
+.spinner.sm {
+  height: 23px;
+  width: 23px;
```

Comment:
This is an unusually odd size in web dev. Why not 24px, which is the size of the child div `18px + 2px margin/side`.